### DISCUSSION TITLE
How can I make it?

### DISCUSSION THREAD

**Discussion by Srijonlahiri** [danny-avila/LibreChat#560] [Troubleshooting]
Please explain step by step what to do to make this app. Is it free to build?

**Comment by danny-avila** [comment]
I'm sorry, but I don't understand your question. This app is completely free to use.

**Comment by fuegovic** [comment]
If you want to run this app, try to follow the instruction in the documentation:
https://github.com/danny-avila/LibreChat/blob/main/docs/install/docker_install.md
